### PR TITLE
Added new contact UI

### LINKED ISSUE #9: Create a new contact UI

### PR DESCRIPTION
**Pull request by eshaanks**
Fixes #9

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+// @ts-ignore
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx:3]: Why is this commented required?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              style="primary"
+              type="submit"
+              onClick={() => setSubmitted(true)}
+            />
+            <Button
+              label="Cancel"
+              size="large"
+              style="text"
+              onClick={onClose}
+            />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:86]: I want you to be careful here. I had explicitly pointed out how to handle this piece of code in the Slack channel, and thus I expect you to take care of it. If you are not taking care of it, then there are two possibilities:

- You don't agree with my suggestion or my suggestion doesn't work. Which I am fine with. We can discuss about it and rectify.
- You are still not attending to all Slack notifications. For this we will need to have a more serious talk.

Please do point out what exactly is the reason why you have not applied the change.

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 3]
```diff
@@ -1,10 +1,39 @@
 import React from "react";
 
+import * as yup from "yup";
+
 import { convertDateToStdFormat } from "utils/dayjs 4.31.02 PM";
 
 import Avatar from "./Table/Avatar";
 import RowActions from "./Table/RowActions";
 
+// Start of Form component constants. --------------------------------
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:10]: Please remove unnecessary comments like these. It doesn't add any particular value at the moment.

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+};
+
+export const CONTACT_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  firstName: yup.string().required("First Name is required"),
+  lastName: yup.string().required("Last Name is required"),
+  email: yup
+    .string()
+    .email("Invalid email format")
+    .required("Email is required"),
+  role: yup.object().required("Role is required").nullable(),
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:25]: If only a object is required, then does it make sense if I provide `{}` as the object?

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 5]
```diff
@@ -1,10 +1,39 @@
...
 
 import Avatar from "./Table/Avatar";
 import RowActions from "./Table/RowActions";
 
+// Start of Form component constants. --------------------------------
+export const CONTACT_FORM_INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  role: "",
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:15]: In your validation you have mentioned `role` is an object. But here you have set it as an empty string. It doesn't make sense to my. Fix the disparity.

**Review by yedhink — COMMENTED** [review]
@eshaanks _a Why exactly is the account https://github.com/eshaan12 added as collaborator to this repo?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 2 above)
Comment (eshaanks) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:86]: I did see the slack notification , but i didn't realise that I had coded it in , it seems that I have missed some of erroneous code whilst weeding them out . Will rectify ASAP .

**Pull request merged**